src/auth/commands.ts

```diff
@@ -1,11 +1,10 @@
 /*!
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Dates are not required. If you find the need to update these, just remove the date.